docs/HowTo/Use-Privacy/Use-FlexiblePrivacy.md

```diff
@@ -42,7 +47,7 @@ the [web3.js-eea library](https://github.com/PegaSysEng/web3js-eea):
     node simpleOnChainPrivacy.js
```

Comment:
this doesn't match web3js-eea - over there it is simpleExample.js - should we rename it there or here?

Reply:
Updated to simpleExample.js